apps/users/forms/profile.py

```diff
@@ -20,17 +25,16 @@
 
 
 class ProfileUpdateForm(forms.ModelForm):
-
     username = forms.EmailField(max_length=254, label="Institution email")
     email = forms.EmailField(max_length=254, label="Personal email (optional)", required=False)
     nationality = forms.ChoiceField(choices=list(zip(NATIONALITIES, NATIONALITIES)))
 
     # Tue:
-    department = forms.ChoiceField(choices=list(zip(TUE_DEPARTMENTS, TUE_DEPARTMENTS)))
-    program = forms.ChoiceField(choices=list(zip(TUE_PROGRAMS, TUE_PROGRAMS)))
+    department = forms.ChoiceField(required=False, choices=list(zip(TUE_DEPARTMENTS, TUE_DEPARTMENTS)))
+    program = forms.ChoiceField(required=False, choices=list(zip(TUE_PROGRAMS, TUE_PROGRAMS)))
 
     # Fontys:
-    study = forms.ChoiceField(choices=list(zip(FONTYS_STUDIES, FONTYS_STUDIES)))
+    study = forms.ChoiceField(required=False, choices=list(zip(FONTYS_STUDIES, FONTYS_STUDIES)))
```

Comment:
Why are these suddenly not required anymore?

Reply:
if I set these to True, the Profile update form cannot be valid because when the user is a TUe student, `study` is empty (in the case of the form, it's an empty string) and when the user is a Fontys student, `department` and `program` is empty.

![image](https://user-images.githubusercontent.com/19555626/125809632-1ab5d5bd-45b2-4fee-92ed-35d11d9ca9a6.png)

I'm not too sure of an alternative solution.

Reply:
ah fair enough yeah, another option would be separate forms for the institution, but i dont think it is really necessary, this works well enough, just maybe add a comment explaining it